citeproc/model.py

```diff
@@ -1387,7 +1383,7 @@ def process(self, item, context=None, **kwargs):
                                          child.calls_variable())
             except VariableError:
                 pass
-        output = [item for item in output if item is not None]
+        output = [item_ for item_ in output if item_ is not None]
```

Comment:
this variable was shadowing the function parameter.  could use a filter instead.